.github/workflows/linters.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: linters
```

Comment:
I'm wondering if it's worth another job or if the unit test job should simply run the `test-fast` target and test all lints + unit tests. The setup time (building container etc.) is probably comparable or longer to the time of actually executing linters + unit tests? I guess i'd prefer just one job for all "blocking checks which do not require running an OpenStack env, and are therefore fast to execute", but it's not a strong opinion, i'd be fine with separate jobs too in case we collectively feel it's a better way.

Reply:
Im checking the actual difference between jobs and workflows, and it seems the workflows are totally independent.

What we can use is to define 2 jobs, one for consistency checks and then the functional tests execution which depends on the consistency check passing.